ui/core-react/src/core-react/badge/DeprecatedBadge.scss

```diff
@@ -2,13 +2,10 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-.nz-widget-tab {
-  &.uifw-badge-new {
-    &.nz-active {
-      &:before {
-        width: calc(100% - 1em) !important;
-        left: 0.5em !important;
-      }
-    }
+.core-badge-deprecatedBadge {
+  transform: rotate(270deg);
+
+  svg {
+    fill: rgba(0, 0, 0, 0.85);
```

Comment:
@GerardasB also do you think it is fine if I used just plain color here? I tried looking for a proper iui variable but all of them change depending on theme or are not visually correct (e.g. too light so contrast is weird). This badge specifically the cross needs to be really black for all themes to not have any weird contrast issues.

Reply:
Looks good to me